config/default.json

```diff
@@ -0,0 +1,3 @@
+{
+    "knowledge_artifact_servers": ["http://pathways.mitre.org:8190/fhir"]
```

Comment:
Can we use the `servers` collection in the data store to store the list of servers instead? Then we could also store additional metadata like "time last fetched", credentials, or anything else we might find useful. Then maybe a separate task would be a command line function to register new servers